examples/algebra/src/main.rs

```diff
@@ -1,5 +1,6 @@
 extern crate alloc;
 
+use openvm;
```

Comment:
if you import `openvm_*` then you don't need `openvm` itself I believe

Reply:
i think if you do `use openvm as _`, clippy wouldn't complain

Reply:
for the sake of uniformity, I put `use openvm as _;` everywhere